### PR TITLE
Tableau live app

### PR DESCRIPTION
**Pull request by nzapponi**
Added sample Tableau Live App

### CODE REVIEW

**Comment by nzapponi** [comment]
@FlikHu ready!

File: examples/tableau/src/components/dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    const vizContainer = useRef<HTMLDivElement | null>(null);
+
+    const filterChangedHandler_ = async (e) => {
+        const worksheet = e.detail.worksheet;
+        const fieldName = e.detail.fieldName;
+        console.log("Filter changed", e);
+        const filters = await worksheet.getFiltersAsync();
+        const filter = filters.find((f) => f.fieldName === fieldName);
+        if (filter) {
+            // rootRecord.setFilter(
```
Comment (haozhi1) [inline examples/tableau/src/components/dashboard.tsx:70]: Why are these commented out?

Comment (nzapponi) [reply examples/tableau/src/components/dashboard.tsx:70]: Because it's future functionality not yet available in the Tableau embedded API.

**Review by haozhi1 — APPROVED** [review]
lgtm. i trust you to have tested the app. is there a working app I can play with?

**Pull request merged**